surface_potential_analysis/kernel/solve.py

```diff
@@ -477,64 +477,6 @@ def get_noise_operators_real_isotropic_stacked_taylor_expansion(
     }
 
 
-def get_diagonal_noise_operators_eigenvalue(
```

Comment:
why was this deleted?

Reply:
below this there is a function called get_noise_operators_daigonal_eigenvalue and i think they are the same?and this deleted function is not called anywhere else

Reply:
Ah so there is - my bad